src/commands/starter/starter.command.ts

```diff
@@ -0,0 +1,121 @@
+import { Message, RichEmbed, User } from 'discord.js';
+import { prefixedCommandRuleTemplate , defaultEmbedColor } from '../../config';
+import {
+    Command,
+    CommandClass,
+    detectStaff
+  } from '../../shared';
+
+@Command({
+    matches: ['starter', 'starterpack', 'starterkit'],
+    ...prefixedCommandRuleTemplate
+})
+
+    /**
+     * @param msg
+     * @param args
+     */
+
+export class StarterCommand implements CommandClass {
+    action(msg: Message, args: string[]) {
+        let whoTag;
+        let client = msg.channel.client;
+
+        // #region Stuff for pinging and sending to pinged users
+        // check for mentioned user
+        if (msg.mentions.users.first() !== undefined  && detectStaff(msg.member)) {
+            whoTag = msg.mentions.users.first();
+            // check if tagged user is a member of of the server
+            if (msg.mentions.members.first() === undefined || msg.mentions.users.first().id !== msg.mentions.members.first().id) {
+                msg.author.send(`<@${whoTag.id}> was not a recognized user.`);
+                return;
+            }
+        }
+        // #endregion
+
+        // if command has no args
+        if (args.length === 1) {
+             whoTag = msg.author;
+        }
+
+        // if someone is actually needing the message
+        if (whoTag !== undefined ) {
```

Comment:
Good `if` check here. If Topher decides to not allow pinging other users, we won't need it, but good type safety anyway.